frontend/src/components/dashboard/create/Create.tsx

```diff
@@ -24,16 +24,27 @@ const Create = () => {
     finishLaterContent,
   } = useContext(FinishLaterModalContext)
   const [isLoading, setLoading] = useState(true)
+  const [isInvalid, setIsInvalid] = useState(false)
 
   useEffect(() => {
-    if (!id) return
+    // Campaign ID is invalid
+    if (!id || !/^[1-9]\d*$/.test(id)) {
+      setIsInvalid(true)
```

Comment:
Another (more imperative?) way to implement this would be to use `history.replace` API, but the current method seems more declarative and React-like.